Dockerfile

```diff
@@ -1,5 +1,12 @@
 FROM cimg/ruby:3.1.1
 
+RUN sudo apt update && \
```

Comment:
do we need this? can we not use a python base image in the ci flow somewhere to grab the schema?

Reply:
Hmmm, 

It means we would need to download the schema using a different image and then save that in CI and then restore that in the kubeconform image when it does validation.

I guess we can do that as well, not sure what is cleaner to have two images run, one to pull schema and second to validate manifests against schema.